src/containers/PageRoot.tsx

```diff
@@ -20,14 +20,45 @@ import { connect } from "react-redux";
 
 const { Header, Content, Sider } = Layout;
 
+const MENU_ITEMS = [
+  {
+    key: "dashboard",
+    name: "Dashboard",
+    icon: "laptop"
+  },
+  {
+    key: "apps",
+    name: "Apps",
+    icon: "code"
+  },
+  {
+    key: "monitoring",
+    name: "Monitoring",
+    icon: "dashboard"
+  },
+  {
+    key: "cluster",
+    name: "Cluster",
+    icon: "cluster"
+  },
+  {
+    key: "settings",
+    name: "Settings",
+    icon: "setting"
+  },
+]
+
+
 interface RootPageInterface extends RouteComponentProps<any> {
   rootElementKey: string;
+  isMobile: boolean;
 }
 
 class PageRoot extends ApiComponent<
   RootPageInterface,
   {
     versionInfo: IVersionInfo | undefined;
+    collapsed: Boolean;
```

Comment:
`Boolean` is a box object and it's completely discouraged. See https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

Use `boolean`

Reply:
ok, got it, first time in typescript